app/components/chat_component.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class='active-admin-chat__conversation-history-container'>
+  <div class='active-admin-chat__conversation-history' data-motion='scroll->load_messages'>
+    <% if conversation && messages.empty? %>
+      <div class='active-admin-chat__conversation-history no-messages'>No messages</div>
+    <% else %>
+      <%= render partial: 'active_admin/chat/messages', locals: { messages: messages } %>
+    <% end %>
+  </div>
+  <div class='active-admin-chat__send-message-container'>
+    <form class='active-admin-chat__send-message-form' data-motion='send_message'>
+      <input id='<%= send_message_id %>' type='text' name='message' placeholder='Send message' />
```

Comment:
needed to set a random id to the input so its value got cleared after submitting a message

Reply:
This seems like a bug in Motion. I'll take a look later this weekend.

Reply:
Upon further investigation, I do think that Motion is working as intended here.

The issue is that since you are not asserting a `value` on this field, the value provided by the user is left in place. This is analgous to what happens with [uncontrolled components](https://reactjs.org/docs/uncontrolled-components.html) in React. Setting `value=""` will cause the field to be cleared every time the component renders. If this is not what you want, you can capture the value that is entered by the user by listening to `change` and re-rendering with that same value.

That said, I don't think the solution you have here is terrible if you don't want to manage the user's message in your state. You are observing the value indirectly through the form and replacing the input each time a message is sent.